backend/transcription/app/transcribe.py

```diff
@@ -1,27 +1,53 @@
-from faster_whisper import WhisperModel
-import time
+from transformers import pipeline
+from langdetect import detect
+import time 
+import traceback
 
+MODEL_NAME3 = "openai/whisper-small"
 model_size = "tiny"
-model = WhisperModel(model_size, compute_type="int8")
+
 
 def transcribe(file_path: str) -> dict:
-    print(f"Transcription of {file_path} started...")
-    start_time = time.time() # TODO only for time print, remove later
-    segments, info = model.transcribe(file_path, beam_size=5)
-    transcription_segments = []
-    for segment in segments:
-        transcription_segments.append({
-            "text": segment.text,
-            "start": segment.start,
-            "duration": segment.end - segment.start
-        })
-    result_dict = {
-        "Detected language": info.language,
-        "Language probability": info.language_probability * 100,
-        "segments": transcription_segments
-    }
-    # TODO only for time print, remove later
-    end_time = time.time() 
-    total_time = end_time - start_time
-    print(f"Transcription of {file_path} finished. Total time: {str(total_time)}")
-    return result_dict
\ No newline at end of file
+    try:
+
+        
+        whisper_pipeline = pipeline(
+            "automatic-speech-recognition",
+            model=MODEL_NAME3,
+            chunk_length_s=30,
+            device="cpu",
+        )
+        #Used to detect the original language of the file
+        To_detect_language = whisper_pipeline(file_path, return_timestamps=True, chunk_length_s=30, batch_size=32, generate_kwargs={"task": "transcribe"})
+        elements = To_detect_language['chunks']
+        Text_To_be_detect = ' '.join([chunk['text'] for chunk in elements])
+        detected_language = detect(Text_To_be_detect)
+        
+        #Transcribe to english
+        transcription = whisper_pipeline(file_path, return_timestamps=True, chunk_length_s=30, batch_size=32, generate_kwargs={ "language": "en", "task": "transcribe"})        
```

Comment:
några rader ovan så detekterar vi språket, ska då skickas in engelska till pipelinen sen? det kanske inte spelar någon roll men vill veta vad ni har testat och upptäckt